app/enterprise/1.3-x/logging.md

```diff
@@ -3,15 +3,9 @@ title: Logging Reference
 toc: false
 ---
 
-## Removing Certain Elements From Your Kong Logs
```

Comment:
Hi @peterskim12 , what's the context for removing this section?

Reply:
it's a duplicate from the section slightly further down.